bellatrix-1/README.md

```diff
@@ -0,0 +1,275 @@
+   just created. Use only `1000000000ustarx`, other amounts will be ignored. STARX is testnet STAR.
+
+   ```sh
+   > starsd add-genesis-account $(starsd keys show <key-name> -a) 1000000000ustarx
+   ```
+
+4. Create the gentx
+
+   ```sh
+   > starsd gentx <key-name> --amount=900000000ustarx --chain-id=bellatrix-1
+   ```
+
+   If all goes well, you will see a message similar to the following:
+
+    ```sh
+    Genesis transaction written to "/home/user/.starsd/config/gentx/gentx-******.json"
+    ```
+
+### Submit genesis transaction
+
+> NOTE: To prevent malicious validators, and to ensure a fair and decentralized launch, the following rules will be enforced:
+> 1. Github accounts must be at least a year old and have history; accounts with little activity may not be accepted.
+> 2. Only one gentx per Github account is allowed
+> 3. We reserve the right to exercise our best judgement to protect the network against Sybil attacks. Preference will be given to validators with a proven track record of validating for other networks.
+
+Submit your gentx in a PR [here](https://github.com/public-awesome/stargaze-testnets)
+
+- Fork [the testnets repo](https://github.com/public-awesome/stargaze-testnets) into your Github account
+
+- Clone your repo using
+
+    ```sh
+    > git clone https://github.com/<your-github-username>/stargaze-testnets
+    ```
+
+- Copy the generated gentx json file to `<repo_path>/bellatrix-1/gentx/`
+
+    ```sh
+    > cd stargaze-testnets
+    > cp ~/.starsd/config/gentx/gentx*.json ./bellatrix-1/gentx/
+    ```
+
+- Commit and push to your repo
+- Create a PR onto https://github.com/public-awesome/stargaze-testnets
+
+### Start your validator node
+
+Once after the genesis is released (_MAR 22 2021 1600 UTC_), follow the instructions below to start your validator node.
+
+#### Genesis & seeds
+
+Fetch `genesis.json` into `starsd`'s `config` directory.
+
+```sh
+> curl https://raw.githubusercontent.com/public-awesome/stargaze-testnets/master/bellatrix-1/genesis.json > $HOME/.starsd/config/genesis.json
+```
+
+Verify you have the correct genesis file:
+
+```sh
+> shasum -a 256 ~/.starsd/config/genesis.json
+fb13172f39d0e888601b828aea104e830aa64c3893ff478194e4d41b2e61f793  genesis.json
+```
+
+Add seed nodes in `config.toml`.
+
+```sh
+> vi $HOME/.starsd/config/config.toml
+```
+
+Find the following section and add the seed nodes.
+
+```sh
+# Comma separated list of seed nodes to connect to
+seeds = ""
+```
+
+```sh
+# Comma separated list of persistent peers to connect to
+persistent_peers = ""
+```
+
+#### Set validator gas fees
+
+You can set the minimum gas prices for transactions to be accepted into your node's mempool. This sets a lower bound on gas prices, preventing spam. Stargaze can accept gas in *any* currency. To accept both ATOM and STARX for example, set `minimum-gas-prices` in `app.toml`.
+
+```sh
+> vi $HOME/.starsd/config/app.toml
+```
+
+```sh
+minimum-gas-prices = "0.025uatom,0.025ustarx"
```

Comment:
ATOM is not available in the faucet so we should leave it out of the testnet imho
```suggestion
minimum-gas-prices = "0.025ustarx"
```